tools/utils.py

```diff
@@ -1,9 +1,11 @@
 import os
 import time
 import traceback
+import json
+import sys
 from json import JSONDecodeError
 from pathlib import Path
-from threading import Thread
+from threading import Thread, currentThread
```

Comment:
currentThread wird hier auch nicht verwendet
```suggestion
from threading import Thread
```

Reply:
Da sind irgendwie viele sachen an stellen drin an denen ich gar nix gemacht hab